core/src/test/scala/com/microsoft/azure/synapse/ml/nbtest/GPUTests.scala

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in project root for information.
+
+package com.microsoft.azure.synapse.ml.nbtest
+
+import com.microsoft.azure.synapse.ml.core.test.base.TestBase
+import com.microsoft.azure.synapse.ml.nbtest.DatabricksUtilities._
+
+import java.util.concurrent.TimeUnit
+import scala.collection.mutable
+import scala.concurrent.Await
+import scala.concurrent.duration.Duration
+
+class GPUTests extends TestBase {
+
+  val clusterId: String = createClusterInPool(GPUClusterName, AdbGpuRuntime, PoolId)
+  val jobIdsToCancel: mutable.ListBuffer[Int] = mutable.ListBuffer[Int]()
+
+  println("Checking if cluster is active")
+  tryWithRetries(Seq.fill(60 * 15)(1000).toArray) { () =>
+    assert(isClusterActive(clusterId))
+  }
+  println("Installing libraries")
```

Comment:
might want to abstract this duplicate code in a function and use for both regular and GPU submission

Reply:
Done.